src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilderInstantiation.cs

```diff
@@ -256,7 +262,62 @@ public override bool IsSubclassOf(Type c)
         {
             throw new NotSupportedException();
         }
-        #endregion
+
+#if MONO
```

Comment:
Instead of ifdefs, it would be more compliant with the repo style convention to have this in partial classes src/mono.